webhooks.md

```diff
@@ -0,0 +1,111 @@
+# Webhooks
+
+Currently supported webhooks for operation on integrations. Action is triggered everytime when hotel updates the integration.
+
+* [Integration created](webhooks.md#integration-created) - Whenever integration is created.
+* [Integration disabled](webhooks.md#integration-disabled) - Whenever integration is disabled.
+* [Integration enabled](webhooks.md#integration-enabled) - Whenever integration is enabled.
+* [Integration deleted](webhooks.md#integration-deleted) - Whenever integration is deleted.
+* [Integration canceled](webhooks.md#integration-canceled) - Whenever integration is canceled.
+* [Integration reinstated](webhooks.md#integration-reinstated) - Whenever integration is reinstated.
+
+## Message
+
+```javascript
+{
+  "Action": "IntegrationCreated",
+  "Data": {
+    "Enterprise": {
+      "Id": "8865aa96-f62d-4f9b-a912-ab2100f60f42",
+      "Name": "Sample Chain Hotel 1"
+    },
+    "Requestor": null,
+    "AccessToken":"9E5E84E9974D4F169662AB2200F27CB1-00B343A0DDA725CACAC028E38E3EABF",
+    "CreatedUtc": "2019-12-13T14:42:52Z",
+    "IsEnabled": true,
+    "Integration": {
+      "Id": "9e5e84e9-974d-4f16-9662-ab2200f27cb1",
+      "Name": "WebhookTEST"
+    }
+  }
+}
+
+```
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Action` | [Action](configuration.md#Action)| required | Type of action |
+| `Data` | [Data](integrations.md#Data) | required | Webhook object data. |
+
+### Data
+
+Structure of  data varies depending on type of the [Action](integrations.md#action):
+
+* `IntegrationCreated` - [Integration created data](integrations.md#integration-created-data)
```

Comment:
Where are these links supposed to go? Are they same as top ones just with `integrations.md` instead of `webhooks.md`?